### ISSUE TITLE
Add a tvOS build target

### ISSUE THREAD

**Issue by didiercf** [wkh237/react-native-fetch-blob#689]
I am currently working on a small tvOS project with ReactNative and trying to read and write files using this library but there doesn't seem to be a build target for tvOS which refrains me from linking the library.

Here are the libraries and versions:

- react-native: 0.54.1
- react-native-fetch-blob: 0.10.8

The error message that shows up is exactly the same as in the issue #251 to which the solution was to link the library, but since I'm working on tvOS and there was no build I think it causes the issue to stay.

I'm not 100% sure the issue comes from the missing build target so if anybody knows the fix to this I'm open to anything!

**Comment by oleksandr-yefremov** [comment]
Correct. There is no tvOS target, and theoretically you could use iOS target, if it did not contain any iOS specific code (like DocumentDir, which is not available on tvOS). 
In [this project](https://github.com/jessesquires/JSQCoreDataKit/issues/61) contributors decided to change DocumentDir to CacheDir on tvOS, which should also work here in react-native-fetch-blob. But I am not sure it's worth the efforts, since [official Blob](https://github.com/react-native-community/react-native-releases/blob/master/CHANGELOG.md#added) has landed and has almost the same feature set.
